augur/curate/titlecase.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
```

Comment:
No need for a shebang at the top of this file since it's not directly executable.

Reply:
fixed in https://github.com/nextstrain/augur/commit/c00156ba7d31026aa89fc5ed68ae5215c09b99bd